src/TLuaInterpreter.cpp

```diff
@@ -12181,9 +12181,9 @@ std::pair<bool, bool> TLuaInterpreter::callLuaFunctionReturnBool(void* pT)
             }
         }
         lua_pop(L, lua_gettop(L));
-        if (error == 0) {
+        if (error == 0) 
             return {true, returnValue};
-        }
+        
```

Comment:
Wait don't we use brackets even for one line if blocks?

Reply:
We do yeah, that's what I was hoping clang-tidy would catch...

Reply:
Qt Creator with the same checks picks up the issue, so it's just this action that's not working right:

![image](https://user-images.githubusercontent.com/110988/109118759-04fb6d80-7744-11eb-86f3-cb00ab1af701.png)

Reply:
Too sad. Raise it upstream?

Reply:
Working on it.